fastlane/Fastfile

```diff
@@ -0,0 +1,60 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+class Constants
+  XCODE_VERSION = "~> 12.4"
+  PROJECT = "Log/Log.xcodeproj"
+  SCHEME = "Log"
+  SWIFTLINT_PATH = "./tools/portable_swiftlint/swiftlint"
+  SWIFTLINT_CONFIG_PATH = "Log/.swiftlint.yml"
+  TEST_DEVICE = "iPhone 12 Pro"
+  TEST_IOS_VERSION = "14.5"
+end
+
+platform :ios do
+  before_all do
+    xcversion(version: Constants::XCODE_VERSION)
+  end
+
+  desc "Lint project"
+  lane :lint_project do |options|
+    reporter = options[:reporter] || 'json'
```

Comment:
What's this `reporter`, where was it set up?

Reply:
Coming as an optional parameter, maybe need to document it in the comment 🙄